sdk/core/azure-core/src/main/java/com/azure/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -584,8 +584,7 @@ private static String serialize(SerializerAdapter serializer, Object value) {
         } else if (value instanceof ExpandableStringEnum<?> || value.getClass().isEnum()) {
             // Enum and ExpandableStringEnum need special handling as these could be wrapping a null String which would
             // be "null" is serialized with JacksonAdapter.
-            String stringValue = String.valueOf(value);
-            return (stringValue == null) ? "null" : stringValue;
+            return String.valueOf(value);
```

Comment:
`String.valueOf` returns `"null"` when the value is null. The old code had a redundant check